### DISCUSSION TITLE
issue with Sklearn in Pycaret

### DISCUSSION THREAD

**Discussion by HooraFakhrmoosavy** [pycaret/pycaret#1840] [Q&A] [answered]
hi
when I am running
from pycaret.classification import *
I am facing this error:
module'sklearn' has no attribute 'pipeline'. I already installed sklearn version 0.23.2 and also I ran this line
from sklearn.pipeline import Pipeline
and it is working to me but not sure why I am facing above error for pycaret.classification. Any idea or comments?

Thanks and looking forward for your help
![image](https://user-images.githubusercontent.com/58143342/141484919-175927b4-5b40-498c-bb2a-2c588cde4cea.png)

**Comment by ngupta23** [comment]
What version of sklearn are you using?

**Comment by moezali1** [comment, marked as answer]
@fakhrmoosavy You must use sklearn==0.23.2

**Comment by HooraFakhrmoosavy** [comment]
Hi

I am using same version but still showing me same error. I am interested to
use your package in my model but I wasnt successful. Do you have any
other solution?

Thanks
Hoora

Thank

**Reply by moezali1** [reply]
Open GitHub issue and give us as much detail as possible. The reproducible Notebook, data, complete traceback.
